cypress/integration/home_page.spec.js

```diff
@@ -0,0 +1,17 @@
+describe('Home page', () => {
+  before(() => {
+    cy.visit('https://www.bbc.com/');
+    cy.getDate();
```

Comment:
@Rabab-Diab this statement is useless since its value is not being saved to a variable. So please delete it.

Reply:
it calls the command where the date is saved in the env. variable .. I removed it from the before block and added it to the test case ..